### ISSUE TITLE
why i can't into the example for more detail

### ISSUE THREAD

**Issue by ChengJunJunJun** [dynamicslab/pysindy#298]
![image](https://user-images.githubusercontent.com/127819698/228765406-fb286202-aea7-4f40-a7b3-49c6ca157ce3.png)

![image](https://user-images.githubusercontent.com/127819698/228765552-c8613689-a640-4bd2-9edb-08f8076c9c27.png)

**Comment by akaptano** [comment]
@znicolaou any idea what is breaking here? For the time being you can find the examples rendered as jupyter notebooks here https://github.com/dynamicslab/pysindy/tree/master/examples

**Comment by Jacob-Stevens-Haas** [comment]
tl;dr: I must have forgotten to finish updating `examples/README.rst`

The `docs/conf.py` includes a setup() function so it can be used as a sphinx extension.  It copies the example notebooks from their locations in the source tree to their locations in `docs/`.  You can see how it interacts with `nbsphinx` in the [build output](https://readthedocs.org/api/v2/build/19021039.txt):
```
Creating file /home/docs/checkouts/readthedocs.org/user_builds/pysindy/checkouts/latest/docs/examples/1_feature_overview/example.ipynb
...
writing output... [ 56%] examples/1_feature_overview/example
```
So the page we're looking for is now in:
https://pysindy.readthedocs.io/en/latest/examples/1_feature_overview/example.html.
I thought I had updated all the links on [examples.rst](https://github.com/dynamicslab/pysindy/blob/d57737f3f7e427ddde7c16b2049b74b6a6298382/examples/README.rst), but apparently I just did notebooks 2 and 5.  The fix is most likely to do the same for all of them.

**Comment by akaptano** [comment]
@Jacob-Stevens-Haas Could you take a stab at fixing this soon? I tried something that seemed to work on my computer to build correctly with sphinx, but it failed on the webpage (readthedocs page) (+ wasted a new version release on it, oops!).

**Comment by ChengJunJunJun** [comment]
> @znicolaou any idea what is breaking here? For the time being you can find the examples rendered as jupyter notebooks here https://github.com/dynamicslab/pysindy/tree/master/examples

**Comment by ChengJunJunJun** [comment]
thank you very much,I am learning it ,it can help me a lot> @znicolaou any idea what is breaking here? For the time being you can find the examples rendered as jupyter notebooks here https://github.com/dynamicslab/pysindy/tree/master/examples

**Comment by Jacob-Stevens-Haas** [comment]
Yeah I can I'll try to get this to work tonight
